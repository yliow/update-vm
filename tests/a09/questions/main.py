from latextool_basic import graph
print(r'''
\begin{center}
%s
\end{center}
''' % graph(yscale=1.5, layout='''
 C
D E
  F
H G I
  J
''',
minimum_size='8mm',
edges='C>D,C>E,E>F,F>G,F>H,F>I,G>J',
A=r"label=$$",
B=r'label=$$',
C=r'label=$$',
D=r'label=$\texttt{*}$',
E=r'label=$\texttt{}$',
F=r'label=$\texttt{}$',
G=r'label=$\texttt{*}$',
H=r'label=$\texttt{*}$',
I=r'label=$\texttt{*}$',
J=r'label=$\texttt{*}$',
edge_label={('A','B'):{'label':r'\texttt{c}'},
('C','D'):{'label':r'\texttt{a}', 'style':'pos=0.8,above,inner sep=3mm,circle'},
('C','E'):{'label':r'\texttt{c}'},
('E','F'):{'label':r'\texttt{a}'},
('F','G'):{'label':r'\texttt{n}'},
('F','H'):{'label':r'\texttt{b}', 'style':'pos=0.5,above,inner sep=1mm,circle'},
('F','I'):{'label':r'\texttt{t}', 'style':'pos=0.5,above,inner sep=1mm,circle'},
('G','J'):{'label':r'\texttt{e}'},
},
))

