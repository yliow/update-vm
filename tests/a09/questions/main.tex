%-*-latex-*-
\input{myassignmentpreamble}
\input{yliow}
\input{ciss350}
\renewcommand\TITLE{Assignment 9}

\begin{document}
\topmatter

\input{preamble.tex}
\newpage Q2. \input{q02.tex}
\newpage Q5. \input{q05.tex}
\end{document}

