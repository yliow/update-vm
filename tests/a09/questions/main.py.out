
\begin{center}

\begin{tikzpicture}
\node at (1,-1.5) [circle,draw,minimum size=8mm] (C) {$$};
\node at (0,-3.0) [circle,draw,minimum size=8mm] (D) {$\texttt{*}$};
\node at (2,-3.0) [circle,draw,minimum size=8mm] (E) {$\texttt{}$};
\node at (2,-4.5) [circle,draw,minimum size=8mm] (F) {$\texttt{}$};
\node at (0,-6.0) [circle,draw,minimum size=8mm] (H) {$\texttt{*}$};
\node at (2,-6.0) [circle,draw,minimum size=8mm] (G) {$\texttt{*}$};
\node at (4,-6.0) [circle,draw,minimum size=8mm] (I) {$\texttt{*}$};
\node at (2,-7.5) [circle,draw,minimum size=8mm] (J) {$\texttt{*}$};
\draw [->,thick] (C) -- (D) node[pos=0.8,above,inner sep=3mm,circle]{\texttt{a}}  ;
\draw [->,thick] (C) -- (E) node[auto,pos=0.5]{\texttt{c}}  ;
\draw [->,thick] (E) -- (F) node[auto,pos=0.5]{\texttt{a}}  ;
\draw [->,thick] (F) -- (G) node[auto,pos=0.5]{\texttt{n}}  ;
\draw [->,thick] (F) -- (H) node[pos=0.5,above,inner sep=1mm,circle]{\texttt{b}}  ;
\draw [->,thick] (F) -- (I) node[pos=0.5,above,inner sep=1mm,circle]{\texttt{t}}  ;
\draw [->,thick] (G) -- (J) node[auto,pos=0.5]{\texttt{e}}  ;

;
\end{tikzpicture}
    
\end{center}

